src/langsmith/evaluation-concepts.mdx

```diff
@@ -3,238 +3,389 @@ title: Evaluation Concepts
+
+Pairwise evaluation works well when directly scoring an output is difficult but comparing two outputs is straightforward. For example, in summarization tasks, choosing the more informative of two summaries is often easier than assigning an absolute score to a single summary.
 
 Learn [how run pairwise evaluations](/langsmith/evaluate-pairwise).
 
+### Reference-free vs reference-based evaluators
+
+Understanding whether an evaluator requires reference outputs is essential for determining when it can be used.
+
+**Reference-free evaluators** assess quality without comparing to expected outputs. These work for both offline and online evaluation:
+- **Safety checks**: Toxicity detection, PII detection, content policy violations
+- **Format validation**: JSON structure, required fields, schema compliance
+- **Quality heuristics**: Response length, latency, specific keywords
+- **Reference-free LLM-as-judge**: Clarity, coherence, helpfulness, tone
+
+**Reference-based evaluators** require reference outputs and only work for offline evaluation:
+- **Correctness**: Semantic similarity to reference answer
+- **Factual accuracy**: Fact-checking against ground truth
+- **Exact match**: Classification tasks with known labels
+- **Reference-based LLM-as-judge**: Comparing output quality to a reference
+
+When designing an evaluation strategy, reference-free evaluators provide consistency across both offline testing and online monitoring, while reference-based evaluators enable more precise correctness checks during development.
+
 ## Experiment
 
-Each time we evaluate an application on a dataset, we are conducting an experiment. An experiment contains the results of running a specific version of your application on the dataset. To understand how to use the LangSmith experiment view, see [how to analyze experiment results](/langsmith/analyze-an-experiment).
+An _experiment_ represents the results of evaluating a specific application version on a dataset. Each experiment captures outputs, evaluator scores, and execution traces for every example in the dataset.
 
 ![Experiment view](/langsmith/images/experiment-view.png)
 
-Typically, we will run multiple experiments on a given dataset, testing different configurations of our application (e.g., different prompts or LLMs). In LangSmith, you can easily view all the experiments associated with your dataset. Additionally, you can [compare multiple experiments in a comparison view](/langsmith/compare-experiment-results).
+Multiple experiments typically run on a given dataset to test different application configurations (e.g., different prompts or LLMs). LangSmith displays all experiments associated with a dataset and supports [comparing multiple experiments](/langsmith/compare-experiment-results) side-by-side.
 
 ![Comparison view](/langsmith/images/comparison-view.png)
 
+Learn [how to analyze experiment results](/langsmith/analyze-an-experiment).
 
 ## Experiment configuration
```

Comment:
imo this isnt conceptual anymore and not something i would want someone new to evals to learn right away. we should move to the Set up evolutions > evaluation techniques section.